cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -811,6 +811,11 @@ impl MachInst for Inst {
         // %reg.
         match self {
             Self::Mov_R_R { is_64, src, dst } if *is_64 => Some((*dst, *src)),
+            Self::XMM_R_R { op, src, dst }
+                if *op == SSE_Op::SSE_Movss || *op == SSE_Op::SSE2_Movsd =>
```

Comment:
At some point this will need to be expanded to include whole-register copies (mova**/movu**).